avogadro/rendering/geometryvisitor.cpp

```diff
@@ -9,6 +9,9 @@
 #include "curvegeometry.h"
 #include "linestripgeometry.h"
 #include "spheregeometry.h"
+#ifdef _3DCONNEXION
+#include "cylindergeometry.h"
```

Comment:
Why is `cylindergeometry.h` needed?

Reply:
Before I moved part of the implementation to tdxextensions.cpp there were an overriden "visit" method for cylinder geometry here. I forgot to remove this include.